index.html

```diff
@@ -0,0 +1,153 @@
+        <ul class="nav-ul Mobile-Menu gap-wide display-none" id="mobile-Menu">
+          <li class='d-right'><button type="button" class="hideOnDesktop-I no-style fc-dark" id="close-menu">&times;</button></li>
+          <li><a class="fc-dark fs-medium fc-dark no-style" href="About.html">About</a></li>
+          <li><a class="fc-dark fs-medium fc-dark no-style" href="#Program">Program</a></li>
+          <li><a class="fc-dark fs-medium fc-dark no-style" href="#Guest-Container">Join</a></li>
+          <li><a class="fc-dark fs-medium fc-dark no-style" href="#Sponsor">Sponsor</a></li>
+          <li><a class="fc-dark fs-medium fc-dark no-style" href="#">News</a></li>
+          <li><a class="fc-orange fs-medium padded-border no-style" href="#">CC Campaign</a></li>
+        </ul>
+      </div>
+    </div>
+  </nav>
+
+  <section class="Headline bg-light">
+    <p class="Headline-title">Hello! Car Enthusiasts</p>
+    <h1 class="h1-mobile Headline-title">Miami Vintage Car Show</h1>
+    <article>
+      A joyful Celebration Believing In the Value of Opennes and Sharing,
+      creating a positive change with people from over 80 countries is 
+      taking place in October, in Korea
+    </article>
+    <h2 class="h2-mobile fc-white">24/06/2023(SAT) ~ 26/06/2023(MON)</h2>
+    <p id="location" class="fc-white"><i class="fa fa-map-marker"></i> Miami Beach, Florida</p>
+  </section>
+
+  <section class="main-program" id="Program">
+    <h3 class="h3 fc-white main-program-title">Main Program
+      <span class="title-underline"></span>
+    </h3>
+
+    <div class="cards">
+    
+      <div class="card">
+        <div class="card-header">
+          <img src="Images/stunt-car.jpg">
+        </div>
+        <div class="card-title">
+          <span class="fc-orange">Car Stunts</span>
+        </div>
+        <div class="card-body">
+          <p class="card-content">Car stunts will be performed by professionals
+          </p>
+        </div>
+      </div>
+
+      <div class="card">
+        <div class="card-header">
+          <img src="Images/logo (2).jpg" class="card-img">
+        </div>
+        <div class="card-title">
+          <span class="fc-orange">Cars Competition</span>
+        </div>
+        <div class="card-body">
+          <p class="card-content">Cars beauty contest will be held and judged by
+            our special guests of the night
+          </p>
+        </div>
+      </div>
+
+      <div class="card">
+        <div class="card-header">
+          <img src="Images/OLAZ6B0.jpg" class="card-img">
+        </div>
+        <div class="card-title">
+          <span class="fc-orange">Winning Price</span>
+        </div>
+        <div class="card-body">
+          <p class="card-content">Winner of the car Competition will be awarded 
+            $20k and a certificate 
+          </p>
+        </div>
+      </div>
+
+      <div class="card">
+        <div class="card-header">
+          <img src="Images/music.jpg">
```

Comment:
- [ ] Please ensure that an image has the `alt` attribute. The alt attribute provides alternative information for an image if a user for some reason cannot view it (because of slow connection, an error in the src attribute, or if the user uses a screen reader) 👍 .